zookeeper-server/zookeeper-server-3.5.6/src/main/java/com/yahoo/vespa/zookeeper/Reconfigurer.java

```diff
@@ -26,6 +27,7 @@
 
     private static final Logger log = java.util.logging.Logger.getLogger(Reconfigurer.class.getName());
     private static final Duration sessionTimeout = Duration.ofSeconds(30);
+    private static final Duration retryReconfigurationPeriod = Duration.ofMinutes(5);
```

Comment:
This seems too long. How about 30 seconds? Typically there will only be one config change and subsequent reconfigs on other nodes will effectively be no-ops.

Reply:
Yes, I have no idea how long this should. But we can try 30 seconds